### ISSUE TITLE
tap to reveal eye icon may be difficult to use on screen readers

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-content-server#6241]
Reported by @devinreams in https://github.com/mozilla/fxa-content-server/issues/6083#issuecomment-393260308

> The "tap to reveal" eye icon that toggles the plaintext password may be considered hard (or potentially impossible?) to toggle with a screen reader like VoiceOver on iOS.

**Comment by vladikoff** [comment]
from mtg: @shane-tomlinson to check this...

**Comment by shane-tomlinson** [comment]
@MarcoZehe confirmed, it's unusable for keyboard users.

**Comment by MarcoZehe** [comment]
Correct, and that goes for keyboard users who are _not_ screen reader users as well. You simply cannot tab out of that field onto thecheckbox that allows you to reveal the password.

Having said that, I noticed that, after entering a password that does meet all criteria, tab moves directly to the Confirm Password field. So when this gets fixed, make sure the checkbox is in the tab order. Best is to not use any tabindex attributes and just let the natural tab order (DOM order) take its course.

**Comment by vbudhram** [comment]
from mtg: Moving this to our general quality backlog in favor of https://github.com/mozilla/fxa-content-server/issues/6311
